python/lsstdesc/pipe/task/coadd_in_cells.py

```diff
@@ -60,12 +56,15 @@ class CoaddInCellsConfig(pipeBase.PipelineTaskConfig,
     """
     seed = Field(
         dtype=int,
-        # default=0,
+        default=0,
         optional=False,
         doc='seed for the random number generator',
     )
-
-    # pass
+    interp_bright = Field(
```

Comment:
I saw a TODO item down below and decided to use this opportunity to make it a config parameter. You might want to double check on the doc.